src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -238,6 +239,15 @@ public boolean hasPart(String partName) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public boolean hasTechnician(int id) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        public void addTechnician(Technician technician) {
```

Comment:
not sure if this will throw a `@Overwrite` error, I assuming it doesn't since you are able to pass gradle checks

Reply:
Will fix